src/pages/trainer/dashboard/page.tsx

```diff
@@ -12,6 +12,7 @@ export class DashboardPage extends React.Component<{}, {}> {
            <br/>
            <Link to="/trainers/evaluation">Go to student evaluation</Link>
            <Link to="/trainers/">Go back to training</Link>
+           <Link to="/trainers/training/edit/1">Edit training content</Link>
```

Comment:
I see this is just harcoded because is something temporal, later on we will have to navigate to training list and from there choose a training (mock stuff)